lib/typescript/render/components/RichCard/Media/index.tsx

```diff
@@ -18,5 +19,6 @@ export const Media = ({height, contentInfo: {altText, fileUrl}}: MediaRenderProp
     className={`${styles.mediaImage} ${
       height === MediaHeight.tall ? styles.tall : height === MediaHeight.medium ? styles.medium : styles.short
     }`}
+    onError={(event: React.SyntheticEvent<HTMLImageElement, Event>) => fallbackImage(event, 'mediaImage')}
   />
```

Comment:
added fallbackImage for all the images inside the render library. This fallback does not work for video, I've created an issue for this #1299